core-web/libs/portlets/edit-ema/portlet/src/lib/store/features/load/withLoad.ts

```diff
@@ -163,14 +152,13 @@ export function withLoad() {
                     )
                 ),
                 /**
-                 *  When the state is "reloaded" outside the editor component, we need to set `isClientReady` to `false`.
-                 *  This is particularly relevant for Single Page Applications (SPA) or client-side applications,
-                 *  as these apps do not have access to the PREVIEW or EDIT modes.
-                 *  If the store updates without resetting `isClientReady` to `false`,
-                 *  returning to the editor component will not trigger a data re-fetch,
-                 *  causing outdated data to persist.
+                 * Reloads the current page to bring the latest content.
+                 * Called this method when after changes, updates, or saves.
+                 *
+                 * @param {Partial<DotPageApiParams>} params - The parameters used to fetch the page asset.
+                 * @memberof DotEmaShellComponent
                  */
-                reload: rxMethod<Pick<UVEState, 'isClientReady'> | void>(
+                reloadCurrentPage: rxMethod<Pick<UVEState, 'isClientReady'> | void>(
```

Comment:
better name for sure.

Reply:
@zJaaal @KevinDavilaDotCMS If you have any ideas, feel free to added it before merging it

Reply:
No I mean this name is better, no change needed.